### PR TITLE
Add price import option

### PR DESCRIPTION
**Pull request by panshul007**

### CODE REVIEW

**Comment by hajoeichler** [comment]
I guess you need to release the importer before and also update the version of the dependency.

**Comment by emmenko** [comment]
@panshul007 provide tests also

**Comment by panshul007** [comment]
The build will pass when it gets the latest version of sphere-product-import

File: src/commands/import.coffee [hunk 1]
```diff
@@ -21,9 +23,10 @@ module.exports = class extends BaseCommand
       .option '-t, --type <name>', 'type of import'
       .option '-f, --from <path>', 'the path to a JSON file where to read from'
       .option '-b, --batch <n>', 'how many chunks should be processed in batches (default: 5)', parseInt, 5
+      .option '-el, --errorLimit <n>', 'maximum number of errors to log (default: 30)', parseInt, 30
```
Comment (emmenko) [inline src/commands/import.coffee]: I don't like this, it's not part of the cli.

**Pull request merged**